plugins/ansiblevault/password.go

```diff
@@ -0,0 +1,61 @@
+package ansiblevault
+
+import (
+	"context"
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+	"os"
+)
+
+func Password() schema.CredentialType {
+	return schema.CredentialType{
+		Name:          credname.Credential,
+		DocsURL:       sdk.URL("https://docs.ansible.com/ansible/latest/cli/ansible-vault.html"),
+		ManagementURL: nil,
+		Fields: []schema.CredentialField{
+			{
+				Name:                fieldname.Password,
+				MarkdownDescription: "Password to use when encrypting or decrypting.",
+				Secret:              true,
+			},
+		},
+		DefaultProvisioner: provision.TempFile(
+			passwordFile,
+			provision.Filename(".ansible-vault"),
+			provision.SetPathAsEnvVar("ANSIBLE_VAULT_PASSWORD_FILE"),
+		),
+		Importer: importer.TryAll(
+			TryPasswordFile(),
+		)}
+}
+
+func passwordFile(in sdk.ProvisionInput) ([]byte, error) {
+	contents := ""
+	if password, ok := in.ItemFields[fieldname.Password]; ok {
+		contents += password
```

Comment:
If the password is not present in the 1Password item (say it's bee deleted) we should return an error to the plugin user explaining this unexpected state.